monitoring/logging.sh

```diff
@@ -251,7 +251,15 @@ RMQMessageRunProp()
 # set the name of the asgs log file
 setSyslogFileName()
 {
-   SYSLOG=`pwd`/${INSTANCENAME}.asgs-${STARTDATETIME}.$$.log
+   local DATETIME=$(date +'%Y-%h-%d-T%H:%M:%S%z')
+   if [[ ! -d $WORK/log ]]; then
+      mkdir -p $WORK/log
```

Comment:
This all looks like a good change, so this is just a note. A lot of times I don't find it necessary to check to see if a directory is created if I am going to create it directly, so I tend to do something like this:

```
# mkdir without checking,
mkdir -p $WORK/log 2> /dev/null
```
However this doesn't cover the cases of:

1. `$WORK/log` exists but is a file
2. `mkdir -p` fails for a reason related to a filesystem issue